golang/cosmos/ante/inbound.go

```diff
@@ -10,83 +10,114 @@ import (
 	swingtypes "github.com/Agoric/agoric-sdk/golang/cosmos/x/swingset/types"
 )
 
+// This AnteDecorator enforces a limit on the size of the Swingset inbound
+// queue by scanning for Cosmos-messages which add Swingset-messages to
+// that queue. Note that when running DeliverTx, inbound messages are
+// staged in the action queue, then transferred to the inbound queue in
```

Comment:
Nit, I'd prefer keeping `actionQueue` instead of `action queue` for grepping.

Reply:
Done.